core/orch_test.go

```diff
@@ -708,27 +700,88 @@ func TestProcessPayment_GivenReceiveTicketError_ReturnsError(t *testing.T) {
 	recipient.AssertNumberOfCalls(t, "RedeemWinningTicket", 2)
 }
 
-func TestProcessPayment_GivenInsufficientCredit_ReturnsErr(t *testing.T) {
+// Check that an Acceptable error increases the credit
+
```

Comment:
Kill space?

Reply:
Fixed in https://github.com/livepeer/go-livepeer/pull/956/commits/f7b0485b60d135b363e3b5d78b6b0b7dadf1e4e9